UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/NoticeToMariners/NoticeToMarinersControllersTest.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners;
+using UKHO.MaritimeSafetyInformation.Web;
+using UKHO.MaritimeSafetyInformation.Web.Controllers;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.NoticesToMariners
+{
+    class NoticeToMarinersControllersTest
```

Comment:
Explicit access modifier please, I suggest `internal`:
```suggestion
    internal class NoticeToMarinersControllersTest
```

Reply:
done